change/@uifabric-styling-1e58bc26-5b98-47e2-a84d-b848bf70c815.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

This should be minor since there's an API change.
```suggestion
  "type": "minor",
```